qa/workunits/cephtool/test.sh

```diff
@@ -986,6 +986,20 @@ function test_mon_mds()
 
   fail_all_mds $FS_NAME
   ceph fs rm $FS_NAME --yes-i-really-mean-it
+
+  # We should be permitted to use an EC pool with overwrites enabled
+  # as the data pool...
+  ceph osd pool set mds-ec-pool debug_white_box_testing_ec_overwrites true --yes-i-really-mean-it
```

Comment:
i think, in long term, maybe we should rely on teuthology to perform the integration tests. and we have `qa/suites/rbd/cli/pool/ec-data-pool.yaml` to enable this feature. but i have no strong opinion on this though.

if @jdurgin is happy, i am fine.